### DISCUSSION TITLE
Discussion 사용법

### DISCUSSION THREAD

**Discussion by Woon-2** [MyeongWoonJang/cpp-study#2] [Announcements]
# Discussion 사용법
`Github Discussion`의 전반적인 사용법을 알리기 위한 글입니다.

## 어떨 때 사용하는가
- `Discussion`은 `질문/답변`, `기안`, `논의`, `잡담`을 위한 용도입니다.
- 프로젝트 내/외적으로 잘 모르겠는 내용에 대한 질문이나 도움이 필요한 부분이 있을 때 사용합니다.
- 협업 규칙, 프로젝트 설계 등 중요한 논의를 할 때 사용합니다.
- 코드나 설계에 대해서 개선 사항이 있을 때 사용합니다.

## 사용 양식
![카테고리, 뉴디스커션](https://user-images.githubusercontent.com/73771162/190853149-ad37c42b-0541-46d1-bcae-ac56b4012f6c.png)
- 왼쪽의 카테고리 필터를 통해 원하는 카테고리의 게시글만 볼 수 있습니다.
- `New Discussion`을 통해 `Discussion`을 생성합니다.
- `Issue`에 비해서 좀 더 자유롭게, 원하는대로 사용하더라도 무관합니다.

___

![셀렉트카테고리](https://user-images.githubusercontent.com/73771162/190853238-f5da9986-4716-4d8a-a70f-9ba924a58916.png)

- `New Discussion` 클릭 시 나타나는 창입니다.
- 카테고리를 설정하고, 게시글을 작성합니다.
- 댓글을 통해 소통합니다.

## 카테고리
![general](https://user-images.githubusercontent.com/73771162/190853338-87a2b813-66fa-4873-8a27-f0041431b788.png)

- 자유게시판 느낌입니다.
- 아래 카테고리에 포함되지 않는 모든 글들은 이 카테고리로 작성해주세요.

___
![ideas](https://user-images.githubusercontent.com/73771162/190853345-48b52489-918e-4a23-ad20-1fd51ee928b8.png)

- 다음과 같은 주제에 사용합니다.   

  - 프로젝트 설계 아이디어
  - 협업 규칙
  - 프로젝트 비전
  - 코드 개선 아이디어
  - 문서화 아이디어
  - 공부에 참고할 만한 좋은 내용

___
![polls](https://user-images.githubusercontent.com/73771162/190853352-05609b7a-289f-489e-98db-d44e49ce797f.png)
- 어떤 사안에 있어서 투표를 진행할 때 사용합니다.

___
![Q A](https://user-images.githubusercontent.com/73771162/190853355-bdf3588f-b558-40bd-b32a-a802ab8363b9.png)
- 프로젝트 내/외적으로 질문 사항이 생겼을 때 사용합니다.
- 코드 리뷰를 통해 발생한 질문 사항은 `Issue`에서 하고, 그 외의 질문 사항을 여기에 적어주세요.

___
![Show Tell](https://user-images.githubusercontent.com/73771162/190853357-bc7252ff-3f1f-4351-b8c1-c0f74540064c.png)
- 여러분의 솜씨를 자랑하세요.

**Comment by Lcat12345** [comment]
확인했습니다!!!
